osmosis-1/osmosis-1.assetlist.json

```diff
@@ -656,6 +656,35 @@
         "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/med.png"
       },
       "coingecko_id": "medibloc"
+    },
+    {
+      "description": "Native Token of Comdex Protocol",
+      "denom_units": [
+        {
+          "denom": "ibc/EA3E1640F9B1532AB129A571203A0B9F789A7F14BB66E350DCBFA18E1A1931F0",
+          "exponent": 0,
+          "aliases": [
+            "ucmdx"
+          ]
+        },
+        {
+          "denom": "cmdx",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/EA3E1640F9B1532AB129A571203A0B9F789A7F14BB66E350DCBFA18E1A1931F0",
+      "name": "Comdex",
+      "display": "cmdx",
+      "symbol": "CMDX",
+      "ibc": {
+        "source_channel": "channel-1",
+        "dst_channel": "channel-87",
+        "source_denom": "ucmdx"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/cmdx.png"
+      },
+      "coingecko_id": "cmdx"
```

Comment:
I don't see comdex listed on coingecko yet

Reply:
Yeah, can we keep it empty for now?

Reply:
```suggestion
```